### PR TITLE
Transform JSON strings recursively

### PR DESCRIPTION
**Pull request by koszti**
## Problem

Transforming JSON strings to object implemented by #20 , but it's working only on top level json strings.

## Proposed changes

Transform JSON strings to objects recursively.

## Types of changes

What types of changes does your code introduce to PipelineWise?

- [x] Bugfix (non-breaking change which fixes an issue)
- [ ] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)
- [ ] Documentation Update (if none of the other choices apply)

## Checklist

- [x] Description above provides context of the change
- [x] I have added tests that prove my fix is effective or that my feature works
- [x] Unit tests for changes (not needed for documentation changes)
- [x] CI checks pass with my changes
- [x] Bumping version in `setup.py` is an individual PR and not mixed with feature or bugfix PRs
- [ ] Commit message/PR title starts with `[AP-NNNN]` (if applicable. AP-NNNN = JIRA ID)
- [ ] Branch name starts with `AP-NNN` (if applicable. AP-NNN = JIRA ID)
- [x] Commits follow "[How to write a good git commit message](http://chris.beams.io/posts/git-commit/)"
- [ ] Relevant documentation is updated including usage instructions

### CODE REVIEW

File: tap_twilio/transform.py [hunk 1]
```diff
@@ -22,14 +22,20 @@ def subresources_to_array(data_dict, data_key):
...
-                try:
-                    data_dict[jsons_key] = json.loads(data_dict[jsons_key])
-                except json.JSONDecodeError:
-                    data_dict[jsons_key] = {'invalid_json': data_dict[jsons_key]}
+    if isinstance(data_dict, dict):
+        # Recursive deserialize JSON string child items
+        for key in data_dict.keys():
+            data_dict[key] = deserialise_jsons_in_dict(data_dict[key], jsons_keys)
+
+        if isinstance(data_dict, dict):
```
Comment (tibortezsla) [inline tap_twilio/transform.py:30]: we don't need this check, since in line 25 we already made sure `data_dict` is a dict, right?

**Review by RobinSindberg — DISMISSED** [review]
approving, but can't really challenge the code.

File: tap_twilio/transform.py (hunk 1 above)
Comment (koszti) [reply tap_twilio/transform.py]: thanks, fixed by https://github.com/transferwise/pipelinewise-tap-twilio/pull/23/commits/145211f2f066e9d50f26dbb86168353826569fab

**Review by tibortezsla — APPROVED** [review]
Looks good, tested it with Robin's PR, and it got all the `task_attributes` and `worker_attributes` data under `event_data`! 
Thanks for the PR 🚀

**Pull request merged**